alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -38,16 +38,15 @@ pub fn is_conpty() -> bool {
     IS_CONPTY.load(Ordering::Relaxed)
 }
 
-#[derive(Clone)]
-pub enum PtyHandle {
-    Winpty(winpty::WinptyHandle),
-    Conpty(conpty::ConptyHandle),
+enum PtyImpl {
```

Comment:
Not super happy with this name. `Impl` feels like it's a struct implementing a trait, but `Pty` is a trait and `PtyImpl` is an enum. Additionally things like `Impl` and `Trait` shouldn't be appended in Rust.

I'm getting Java flashbacks from my dayjob here, so I think we should probably rename this to me more rusty

TODO: Suggest better name.